### ISSUE TITLE
Illustration for expected download of sample with CITE-seq as AnnData

### ISSUE THREAD

**Issue by allyhawkins** [AlexsLemonade/scpca-docs#152]
We need to create a new illustration for the expected download structure when downloading a sample _with CITE-seq_ as AnnData.

Below is the expected structure copied from https://github.com/AlexsLemonade/ScPCA-admin/issues/595#issuecomment-1644330881

```
sample_id
├── README.md
├── SCPCL000000_filtered_adt.hdf5
├── SCPCL000000_filtered_rna.hdf5
├── SCPCL000000_processed_adt.hdf5
├── SCPCL000000_processed_rna.hdf5
├── SCPCL000000_qc_report.html
├── SCPCL000000_unfiltered_adt.hdf5
├── SCPCL000000_unfiltered_rna.hdf5
└── single_cell_metadata.tsv
```

**Comment by dvenprasad** [comment]
How does this look?

<img width="350" alt="anndata-CITE-seq-sample-download-folder" src="https://github.com/AlexsLemonade/scpca-docs/assets/15315514/df3ac2e4-ef28-48ba-a164-a383d472f501">

**Comment by allyhawkins** [comment]
All look good thank you!!

**Comment by allyhawkins** [comment]
Closed by #161
